purview/api/v1.py

```diff
@@ -75,24 +76,28 @@ def get_resource_story(resource, uid):
 
     results_unordered = {}
     if forward_query:
+        print("Forward Query:", forward_query)
         results_unordered = query_neo4j(forward_query)
 
     if backward_query:
         results_unordered.update(query_neo4j(backward_query))
 
-    results = []
+    results = OrderedDict()
```

Comment:
Does this need to be an `OrderedDict`?

Reply:
I kept this so that the order is always 'data' and then 'meta'.

Reply:
Okay, it's not necessary but I don't have a problem with it either. You can keep it as is.